<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>色区容限调整</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/image/color.png</normaloff>:/image/color.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget {
	background-color: rgb(255, 255, 255);
}
QLineEdit {
	border:  none;
	border-bottom: 1px solid grey;
	padding: 2px;
}

QLineEdit:hover {
	 border-bottom: 2px solid lightblue;
}
QLabel {
	font: 15px &quot;楷体&quot;;
}
#label  {
	color: green;
}
#label_2  {
	color: red;
}
QLineEdit {
	font: 14px &quot;楷体&quot;;
}
QComboBox {
    border: 1px solid #999;
    border-radius: 5px;
    background-color: #fff;
    padding: 5px;
    font: 14px &quot;楷体&quot;;
    color: #333;
}
QComboBox::drop-down {
    border: none;
    background: transparent;
}

QComboBox::down-arrow {
	image: url(:/image/arrow.png);
    width: 20px;
    height: 20px;
	margin-right: 10px;
}
QComboBox::item {
    padding: 5px;
}
QComboBox::item:selected {
    background-color: #ff6a00;
    color: white;
}
QScrollBar:vertical {
    width: 0px;
    background: transparent;
}
QScrollBar:horizontal {
    height: 0px;
    background: transparent;
}
#frame {
	border-image: url(:/image/sketch.png);
}</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>原始色区</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>调整色区</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="p1x">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3804</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p1y">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>10</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3710</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p2x">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3739</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p2y">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>40</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3758</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p3x">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3833</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p3y">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>70</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3885</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p4x">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>100</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3897</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p4y">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>100</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>0.3836</string>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="p1x_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p2x_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>170</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p4x_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>230</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p4y_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>230</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p1y_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>140</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p3x_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>200</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p2y_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>170</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="p3y_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>200</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p2</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p3</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>110</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p4</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>150</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>180</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>p2</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>210</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p3</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>240</y>
           <width>31</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>p4</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>300</y>
           <width>161</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>0.0000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0005</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0006</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0007</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0008</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0009</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0010</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0011</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0012</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0013</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0014</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0015</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0016</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>360</y>
           <width>161</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>0.0000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0001</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0002</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0003</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0004</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0005</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0006</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0007</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0008</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0009</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0010</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0011</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0012</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>330</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>短轴方向调整</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>270</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>长轴方向调整</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>281</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>390</y>
           <width>281</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>400</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>色区坐标点图示</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>430</y>
           <width>281</width>
           <height>161</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>容限调整控制区</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
